From 8c72707d38ca6d358b20eac7909e05670c3c5e95 Mon Sep 17 00:00:00 2001
From: asfand987 <asfand987.ak@gmail.com>
Date: Mon, 14 Sep 2020 15:35:52 +0100
Subject: [PATCH] Fixed typo in main method of client.py

---
 client.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/client.py b/client.py
index 0dd84af..2be4926 100644
--- a/client.py
+++ b/client.py
@@ -60,4 +60,4 @@ if __name__ == "__main__":
 			prices[stock] = price
 			print "Quoted %s at (bid:%s, ask:%s, price:%s)" % (stock, bid_price, ask_price, price)
 
-		print "Ratio %s" % getRatio(price['ABC'], price['DEF'])
+		print "Ratio %s" % getRatio(prices['ABC'], prices['DEF'])
-- 
2.17.1

